packages/Search/src/details/DTK_DetailsTreeTraversal.hpp

```diff
@@ -113,7 +113,7 @@ struct TreeTraversal
         // Nodes with a distance that exceed that radius can safely be
         // discarded. Initialize the radius to infinity and tighten it once k
         // neighbors have been found.
-        double radius = KokkosHelpers::ArithTraits<double>::infinity();
+        double radius = KokkosExt::ArithmeticTraits::infinity<double>::value;
```

Comment:
Is the change from a static function in a stateless class to a struct with a `constexpr` enable performance or some template metaprogramming or was the change for aesthetics?

Reply:
The design of `Kokkos::ArithTraits` is inspired from `std::numeric_limits` but the standard library is currently looking at moving away from such monolithic class templates.  I feel this will be easier to maintain and extend.

Reply:
@sslattery What happen is I had contributed to `Kokkos::ArithTraits` the member function `infinity()` but we were not able to use it yet because we use an older version of Trilinos.  That's why I had define `ArithTraits` in our own namespace.  But now I want to drop the dependency on KokkosKernel.  We had forgotten to register it with TriBITS but it somehow worked because we depend on Tpetra...